### PR TITLE
Update: Reference messaging bridge pattern when refactoring to Azure Service Bus

### PR DESCRIPTION
**Pull request by kbaley**
For "lift and shift" migrations, when refactoring to Azure Service Bus, it can reduce the risk to migrate endpoints one at a time instead of all at once. This change makes reference to the "messaging bridge" pattern as a means of accomplishing that

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers - Please review the proposed changes. If you want to move forward with the content, add a sign-off.
		
#label:"pending-content-team/business-approval"
Cc: @MicrosoftDocs/public-repo-pr-review-team

**Comment by BryanLa** [comment]
Hi @kbaley - thanks very much for your contribution. Can you also work this implementation into your addition, to cite an example: https://github.com/Microsoft/Microsoft-Message-Bridge ?

**Comment by BryanLa** [comment]
#label:"cact-review-in-progress"

File: docs/industries/retail/migrate-ecommerce-solution.md
Comment (kbaley) [inline docs/industries/retail/migrate-ecommerce-solution.md:97]:
```suggestion
For app services, you can take advantage of various Azure services. For example, your existing infrastructure may handle a customer order by placing messages into a queue like [RabbitMQ](https://www.rabbitmq.com/). (For example, one message is to charge the customer, a second is to ship the order.) When rehosting, you put RabbitMQ in a separate VM. During refactoring, you can add a [Service Bus](/azure/service-bus-messaging/service-bus-queues-topics-subscriptions?WT.mc_id=retailecomm-docs-scseely) queue or topic to the solution. At this point, you can rewrite your RabbitMQ code and stop using the VMs that served the queuing functionality. If it's not feasible to rewrite all your code at once, you can use patterns such as the [messaging bridge](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessagingBridge.html) to bridge the gap between messaging queues. This allows you to migrate your endpoints one-by-one rather than all at once. Either way, when all the endpoints have been ultimately moved to Azure Service Bus, this replaces a set of VMs with an always-on message queue service for a lower cost. Other app services can be found in the Azure Portal.

Two implementations of the Messaging Bridge Pattern are the [Microsoft Message Bridge](https://github.com/Microsoft/Microsoft-Message-Bridge) and the [NServiceBus Transport Bridge](https://docs.particular.net/nservicebus/bridge/). 
```

**Comment by kbaley** [comment]
@BryanLa Done. I included another example implementation of it as well.

**Comment by BryanLa** [comment]
> @BryanLa Done. I included another example implementation of it as well.

Great, thank you @kbaley . We're planning to have a related discussion re: our patterns library in a couple of weeks, and possibly adding this as a new pattern article. We may move some of the implementation ideas to that article, and point to it from this one. Just wanted to give you a heads up that we'll probably hold off on this merge until we resolve that part.

**Comment by kbaley** [comment]
@BryanLa That makes sense. I thought the same thing and have started a Message Bridge article already. If you like, I can post that up as a starting point for you. I usually find it easier to edit an existing draft than to start from scratch.

**Comment by kbaley** [comment]
@BryanLa Actually, I see that a PR is [already started](https://github.com/MicrosoftDocs/architecture-center/pull/4068) for the new article by my colleague, Szymon.

**Comment by ckittel** [comment]
Anything new to report on this, @BryanLa.  Can this PR be closed?

**Comment by BryanLa** [comment]
@ckittel - still discussing this one and #4068. I just sent you an email update..

**Comment by BryanLa** [comment]
@kbaley - thanks again for your contribution and patience. 

We discussed both this PR and [PR 4068](https://github.com/MicrosoftDocs/architecture-center/pull/4068) today. The plan of action is to:
- merge your contribution as-is (without citing the other/older implementations)
- review and work with SzymonPobiega on PR 4068 over the next 2 months, to draft the messaging bridge pattern article in our patterns library
- go back and update [Migrate your e-commerce solutions to Azure](https://learn.microsoft.com/azure/architecture/industries/retail/migrate-ecommerce-solution#refactor) and other articles, to point to the new messaging bridge pattern article

cc: @ckittel @RobBagby

**Comment by BryanLa** [comment]
CACT #sign-off
#remove-label:"do-not-merge"
#label:"ready-to-merge"

File: docs/industries/retail/migrate-ecommerce-solution.md
Comment (Jak-MS) [inline docs/industries/retail/migrate-ecommerce-solution.md:97]:
```suggestion
For app services, you can take advantage of various Azure services. For example, your existing infrastructure may handle a customer order by placing messages into a queue like [RabbitMQ](https://www.rabbitmq.com/). (For example, one message is to charge the customer, a second is to ship the order.) When rehosting, you put RabbitMQ in a separate VM. During refactoring, you can add a [Service Bus](/azure/service-bus-messaging/service-bus-queues-topics-subscriptions?WT.mc_id=retailecomm-docs-scseely) queue or topic to the solution. At this point, you can rewrite your RabbitMQ code and stop using the VMs that served the queuing functionality. If it's not feasible to rewrite all your code at once, you can use patterns such as the [messaging bridge](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessagingBridge.html) to bridge the gap between messaging queues. This allows you to migrate your endpoints one-by-one rather than all at once. Either way, when all the endpoints have been ultimately moved to Azure Service Bus, this replaces a set of VMs with an always-on message queue service for a lower cost. Other app services can be found in the Azure portal. 
```

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**
